README.md

```diff
@@ -48,100 +48,96 @@ Your website/app must present information in ways people can recognise and use,
+#### Help people navigate and find content
 
-* [WEB ONLY] Give keyboard users a way to go directly to the page's main content. [More about 2.4.1]()
+* [Skip to main content:]() Give keyboard and screen reader users a way to go directly to the page's main content. [WEB ONLY]
 
-* [WEB ONLY] Give every page a unique and helpful title that indicates the purpose of the page. [More about 2.4.2]()
+* [Page titled:]() Give every page a unique and helpful `title` that indicates the purpose of the page. [WEB ONLY]
 
-* [WEB ONLY] Make sure that interactive elements receive focus in an order that makes sense. [More about 2.4.3]()
+* [Heading and labels:]() Provide headings and form labels that will help people find content and complete tasks.
 
-* Make sure the purpose of a link is obvious from its link text alone, or its link text in context. [More about 2.4.4](guidelines/2.4.4.md)
+* [Focus order:]() Make sure that elements receive focus in an order that makes sense when people use a keyboard or a screen reader.
 
-* Give people different ways of finding each page (like searching or browsing links), unless the page is a step in a process. [More about 2.4.5]()
+* [Focus visible:]() Make sure that keyboard-only users can clearly see which element has the keyboard focus on a page.
 
-* Provide headings and form labels that will help people find content and complete tasks. [More about 2.4.6]()
+* [Link purpose:]() Make sure the purpose of every link is obvious from the link text alone (or the link text plus associated content – if screen readers recognise the association).
 
-* Make sure that keyboard-only users can clearly see which element has the keyboard focus on a page. [More about 2.4.7]()
+* [Multiple ways to find a page]() Give people different ways of finding each page (like searching or browsing links), unless the page is a step in a process.
 
-* Make sure that keyboard focus indicators are clearly visible. [More about 2.4.7]()
+#### Give people enough time to read and use content
 
-* Do not require complex gestures or using more than one finger to do things. [More about 2.5.1]()
+* [Timing adjustable:]() If there is any time limit (like a session timeout), give people an easy way to extend it.
 
-* Controls do not make things happen as soon as they are touched. [More about 2.5.2]()
+* [Stop visual distractions:]() If any blinking, scrolling, animation or frequent updating happens automatically and last more than 5 seconds, give people a way to stop it.
 
-* When an interactive element has a visible label, make sure that the element's accessible name matches (or at least includes) what's on the label. [More about 2.5.3]()
+#### Do not cause seizures
 
-* If a feature uses shaking or tilting, make sure that it can also be activated in another way. And that responses to these motions can be turned off. [More about 2.5.4]()
+* [No three flashes:]() Do not show anything that flashes more than three times a second.
```

Comment:
Not sure "No three flashes" reads super well. Maybe "No extended flashing" or something similar?